customer-stack/customer-stack.yml

```diff
@@ -13,17 +13,14 @@ Parameters:
     AllowedPattern: "^(arn:aws[a-z-]*:kms:[a-z0-9-]*:[0-9]{12}:key\\/.+)?$"
     Default: ""
     Description: "(Optional) ARN of the AWS Key Management Service key that Amazon SageMaker uses to encrypt job outputs. The KmsKeyId is applied to all outputs."
-# Future consideration
-#  vpcSecurityGroupIds:
-#    Type: "String"
-#    AllowedPattern: "^([-0-9a-zA-Z]+(,\\s*[-0-9a-zA-Z]+)*)?$"
-#    Default: ""
-#    Description: "(Optional) Comma delimited list of security group ids for VPC configuration"
-#  vpcSubnetIds:
-#    Type: "String"
-#    AllowedPattern: "^([-0-9a-zA-Z]+(,\\s*[-0-9a-zA-Z]+)*)?$"
-#    Default: ""
-#    Description: "(Optional) Comma delimited list of subnet ids for VPC configuration"
+  vpcSecurityGroupIds:
+    Type: "String"
+    Default: ""
+    Description: "(Optional) Comma delimited list of security group ids for VPC configuration"
+  vpcSubnetIds:
+    Type: "String"
```

Comment:
Curious why not add an `AllowedPattern` to vpcSecurityGroupIds and vpcSubnetIds? If there's any kind of restrictions on the ids, then we could benefit with even a little bit of validation here.

Reply:
Added `AllowedPattern` for subnetids, securitygroups and snowflake vpc id.